src/rviz/tool.h

```diff
@@ -192,6 +192,7 @@ class RVIZ_EXPORT Tool : public QObject
 
 Q_SIGNALS:
   void close();
+  void nameChanged(Tool* tool);
```

Comment:
Please change the argument to `const QString&`, passing the new name. You can always access the sender of a signal by other means, namely [`QObject::sender()`](https://doc.qt.io/archives/qt-4.8/qobject.html#sender).